content/en/docs/zero-code/obi/performance.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
This is probably outdated and refers to OBI unsupported metrics like process metrics. I'd just remove this file and, or leave it empty with a "TO DO" tag for measuring performance impact later.